From d33bed3836c54192f1b51fe0aeeee202342088a0 Mon Sep 17 00:00:00 2001
From: Li Jun <jun.li@freescale.com>
Date: Thu, 26 Nov 2015 17:58:56 +0800
Subject: [PATCH] MLK-11896-5 usb: chipidea: imx: add break for switch case of
 OTG ADP

Fix the coverity check complains on break missing of switch cases.

Acked-by: Peter Chen <peter.chen@freescale.com>
Signed-off-by: Li Jun <jun.li@freescale.com>
---
 drivers/usb/chipidea/ci_hdrc_imx.c | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/drivers/usb/chipidea/ci_hdrc_imx.c b/drivers/usb/chipidea/ci_hdrc_imx.c
index 66dfcfc..4fb9bbb 100644
--- a/drivers/usb/chipidea/ci_hdrc_imx.c
+++ b/drivers/usb/chipidea/ci_hdrc_imx.c
@@ -293,16 +293,20 @@ static int ci_hdrc_imx_notify_event(struct ci_hdrc *ci, unsigned event)
 	case CI_HDRC_IMX_ADP_IS_PROBE_INT:
 		if (data->usbmisc_data)
 			return imx_usbmisc_adp_is_probe_int(data->usbmisc_data);
+		break;
 	case CI_HDRC_IMX_ADP_IS_SENSE_INT:
 		if (data->usbmisc_data)
 			return imx_usbmisc_adp_is_sense_int(data->usbmisc_data);
+		break;
 	case CI_HDRC_IMX_ADP_SENSE_CONNECTION:
 		if (data->usbmisc_data)
 			return imx_usbmisc_adp_sense_connection(
 						data->usbmisc_data);
+		break;
 	case CI_HDRC_IMX_ADP_ATTACH_EVENT:
 		if (data->usbmisc_data)
 			return imx_usbmisc_adp_attach_event(data->usbmisc_data);
+		break;
 	case CI_HDRC_IMX_TERM_SELECT_OVERRIDE_FS:
 		if (data->usbmisc_data)
 			return imx_usbmisc_term_select_override(
-- 
1.8.0

